CONTRIBUTING.md

```diff
@@ -317,6 +317,8 @@ GitHub provides reactions as a simple and quick way to provide feedback to pull
 
 Pull request reviews should include a list of what has been reviewed in a comment, so other reviewers and mergers can know the state of the review.
 
+Pull request reviews that only nitpick things that are easily fixed in treewides cause unnecessary friction and are better left unposted. Nitpicks may however accompany blockers.
```

Comment:
```suggestion
Pull request reviews that only nitpick things that are stylistic in nature or easily fixed in treewides cause unnecessary friction and are better left unposted. Nitpicks may however accompany blockers.
```
?

Reply:
Perhaps "opinionated" would be a better word as we do want to allow change requests for small things that have tangible technical merit for the change at hand but not "you should use pattern x here rather than pattern y" because you prefer pattern y.